### PR TITLE
Add noip package

### PR DESCRIPTION
**Pull request by iand675**
This adds a package to nixpkgs for a small dynamic DNS daemon that works with www.noip.com accounts.

### CODE REVIEW

File: pkgs/tools/networking/noip/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  src = fetchurl {
+    url = http://www.noip.com/client/linux/noip-duc-linux.tar.gz;
+    sha256 = "82b9bafab96a0c53b21aaef688bf70b3572e26217b5e2072bdb09da3c4a6f593";
+  };
+
+  makeFlags = [ "PREFIX=\${out}" ];
+  installPhase =
+    ''
+      if [ ! -d "$out/bin" ]; then mkdir -p $out/bin;fi
```
Comment (lethalman) [inline pkgs/tools/networking/noip/default.nix]: You can drop this if statement. Just `mkdir -p $out/bin`.

Comment (iand675) [inline pkgs/tools/networking/noip/default.nix]: Good point, that's what I get for copying and pasting from the original Makefile.

**Comment by iand675** [comment]
@lethalman fixed code review comment

**Comment by lethalman** [comment]
Built locally, thanks.

**Pull request merged**